### DISCUSSION TITLE
Mark item in Favorite list to defines as standard/TopFavorite

### DISCUSSION THREAD

**Discussion by s25a** [punxaphil/custom-sonos-card#307] [General]
Hi, thanks a lot for this great integration :-) Just found it and really happy with this. Is there a possibility to mark a certain item in the FavoriteList to use it as the "Standard/Top" Favorite? This should help all users in the house to easy select the favorite that will be used when starting the music from a switch. I found the TopFavorite setting that prefer one item and show it at the Top of the List. However this also have to be hardcoded in the settings as far as I understand. Any idea how to realize this? Thanks S

**Comment by punxaphil** [comment]
Does everyone use the same device? Otherwise I don't see a solution. This is just a frontend so it will have to be save in the browsers storage.

**Comment by POWVorti** [comment]
This goes together with my question. I tried to filter my favorites using Templating. 
https://community.home-assistant.io/t/templating-customizing-favorites-in-sonos-custom-card/758191
But I was not able to make it work... Do you support templates with your card?

**Comment by punxaphil** [comment]
Templating is not support I am afraid.
If you want different top favorites for different users, create different dashboards.
